nsc/review/migrations/0022_alter_historicalreview_published_and_more.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 5.0.14 on 2025-10-30 16:16
```

Comment:
Should we remove these migrations from this change, as we aren't sure how to test them? And we don't need to touch this model?